docs/images-integration.md

```diff
@@ -0,0 +1,48 @@
+# Images integration
+
+Wagtail AI integrates with the image edit form to provide AI-generated descriptions to images. The integration requires a backend that supports image descriptions, such as [the OpenAI backend](../ai-backends/#the-openai-backend).
+
+## Configuration
+
+1. In the Django project settings, configure an AI backend, and a model, that support images. Set `IMAGE_DESCRIPTION_BACKEND` to the name of the model:
+   ```python
+   WAGTAIL_AI = {
+       "BACKENDS": {
```

Comment:
**Suggestion:** This seems pretty clear but I'm not sure it's obvious enough from this example that you need to configure two separate backends if you want both image description and text completion features.

Do you think specifying a `IMAGE_DESCRIPTION_MODEL_ID` config setting (with a default value) on the `OpenAIBackend` would make it less of a burden to configure?

Reply:
I've extended the docs to have an example with two backends. I don't think it would help to add another config, and have the backend switch models on the fly, that feels confusing.

Reply:
This seems OK, I am still a bit stuck on the idea that the simplest configuration to get all the fancy Wagtail AI features should be:

```python
WAGTAIL_AI = {
    "BACKENDS": {
        "default": {
            "CLASS": "wagtail_ai.ai.openai.OpenAIBackend",
        },
    },
}
```

but appreciate that we have to balance technical complexity here too, and your configuration doesn't necessarily prevent us from doing that in the future.

We're still asking a lot of a user who might know what AI can do for them, but doesn't understand what models to use in what situations so will just be copying examples from the docs, but again I'm sure we can review at another time.

Reply:
> We're still asking a lot of a user who might know what AI can do for them, but doesn't understand what models to use in what situations so will just be copying examples from the docs, but again I'm sure we can review at another time.

I agree it's not the best experience, and ideally they _should_ be able to just specify a default backend with minimal configuration. But, keeping in mind that the model we're suggesting for images is called `gpt-4-vision-preview`, I think we can hold off until the functionality becomes part of the mainline offering.